src/rust/bitbox02-noise/src/noise_xx.rs

```diff
@@ -213,6 +213,7 @@ impl<R: Random32> State<R> {
                 let encrypted_len = msg.len() + 16;
                 // Make space for result.
                 out.resize(start + encrypted_len, 0);
+                // This also adds the MAC.
```

Comment:
please consider adding another comment on the line 213 above `... + 16; // extra 16 bytes for MAC`.

Reply:
Done